app/mailers/application_mailer.rb

```diff
@@ -1,5 +1,9 @@
-class ApplicationMailer < ActionMailer::Base
+class ApplicationMailer < Mail::Notify::Mailer
   def default_url_options
     { host: Plek.new.external_url_for("content-publisher") }
   end
+
+  def template_id
+    @template_id ||= ENV.fetch("GOVUK_NOTIFY_TEMPLATE_ID", "fake-test-template-id")
```

Comment:
```suggestion
    ENV.fetch("GOVUK_NOTIFY_TEMPLATE_ID", "fake-test-template-id")
```

I don't think we really gain anything from this being a memo'd variable?